driftbase/api/matches.py

```diff
@@ -277,11 +290,14 @@ def get(self, args):
 
             matches = []
             for match_row in matches_result.items:
-                # Service role gets a list of ORM object, but the client (which selects specific columns) gets an SQLAlchemy Row object
-                if is_service:
+                if isinstance(match_row, Row):
+                    match_record = match_row.Match.as_dict()
+                    if hasattr(match_row, 'is_winner'):
+                        match_record["is_winner"] = match_row.is_winner
+                elif isinstance(match_row, Match):
                     match_record = match_row.as_dict()
                 else:
-                    match_record = match_row._asdict() # The Row object has an _asdict() method that *isn't* private/protected. It's part of the NamedTuple API
+                    match_record = match_row._asdict()
```

Comment:
will this ever be visited? as its previous intention was when `match_row` is a `Row` object